src/fileio/datafile.cxx

```diff
@@ -68,11 +66,8 @@ Datafile::Datafile(Datafile &&other) :
   enabled(other.enabled), shiftOutput(other.shiftOutput), file(other.file.release()), int_arr(other.int_arr),
   BoutReal_arr(other.BoutReal_arr), f2d_arr(other.f2d_arr),
   f3d_arr(other.f3d_arr), v2d_arr(other.v2d_arr), v3d_arr(other.v3d_arr) {
-  filenamelen=other.filenamelen;
-  filename=other.filename;
-  other.filenamelen=0;
-  other.filename=nullptr;
-  other.file = nullptr;
```

Comment:
I wasn't sure whether after a move the old pointer is set to `nullptr` is this guaranteed by the standard (and done by compilers)?